components/style/thread_state.rs

```diff
@@ -53,7 +53,9 @@ thread_local!(static STATE: RefCell<Option<ThreadState>> = RefCell::new(None));
 pub fn initialize(x: ThreadState) {
     STATE.with(|ref k| {
         if let Some(ref s) = *k.borrow() {
-            panic!("Thread state already initialized as {:?}", s);
+            if x != *s {
```

Comment:
What is this change for?

Reply:
See the second commit description.

Reply:
(TL;DR: Unbusts running multiple benchmarks)

Reply:
OK.  (I realize I was reviewing in the Files Changed tab instead of the Commits tab, so I missed the separate commit messages...)